hazelcast/src/main/java/com/hazelcast/client/config/impl/AbstractQueryCacheConfigBuilderHelper.java

```diff
@@ -60,32 +60,29 @@ protected String getTextContent(Node node) {
     }
 
     protected void populateQueryCacheConfig(QueryCacheConfig queryCacheConfig,
-                                            Node childNode, String textContent, String nodeName) {
+                                            Node childNode, String nodeName) {
```

Comment:
Here, it's just actually to simplify things. There's no point in passing both `Node` and the content because the content is in the `Node`.

When it comes to _eager_, it just means we should not fetch the value when we're not sure it's needed. And in many places, we'd fetch the value first, and only then check if they key matches some of the existing config properties